open_dread_rando/files/levels/s010_cave.lc.lua

```diff
@@ -376,7 +377,7 @@ end
 
 
 function s010_cave.OnProtoEmmyCantClimbCutsceneStart()   
-    
+  Game.SetItemAmount(Game.GetPlayerName(), "ITEM_MORPH_BALL", MORPH_BALL_AT_PROTO_EMMY)
```

Comment:
Make this a `math.max` between MORPH_BALL_AT_PROTO_EMMY and current value, just in case.